.github/actions/cancel_azureml_jobs/cancel_aml_jobs.py

```diff
@@ -0,0 +1,42 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import os
+
+from azureml._restclient.constants import RunStatus
+from azureml.core import Experiment, Run, Workspace
+from azureml.core.authentication import ServicePrincipalAuthentication
+
+
+def cancel_running_and_queued_jobs() -> None:
+    print("Authenticating")
+    auth = ServicePrincipalAuthentication(
+        tenant_id='72f988bf-86f1-41af-91ab-2d7cd011db47',

```

Comment:
Do we normally keep the `tenant_id` in clear text?

Reply:
This is public knowledge, in a way. There are web pages that use MS pages to demonstrate how to read a tenant ID, for example.